digdag-cli/src/main/java/io/digdag/cli/Main.java

```diff
@@ -45,12 +44,18 @@
     private final io.digdag.core.Version version;
     private final PrintStream out;
     private final PrintStream err;
+    private final Map<String, String> env;
+
+    public Main(io.digdag.core.Version version, PrintStream out, PrintStream err) {
+        this(version, out, err, System.getenv());
+    }
 
-    public Main(io.digdag.core.Version version, PrintStream out, PrintStream err)
+    public Main(io.digdag.core.Version version, PrintStream out, PrintStream err, Map<String, String> env)
```

Comment:
You put `env` right before `out` and `err` for other cli classes. So putting `env` before `out` here in the same way would be better.